config/checkstyle_checks.xml

```diff
@@ -222,7 +222,9 @@
     <module name="IllegalCatch">
         <property name="illegalClassNames" value="java.lang.Exception, java.lang.Throwable, java.lang.RuntimeException, java.lang.NullPointerException"/>
     </module>
-    <module name="IllegalInstantiation"/>
+    <module name="IllegalInstantiation">
+        <property name="classes" value="SAXException, SAXParseException, ConversionException, ParseCancellationException, RecognitionException, TokenStreamException, TokenStreamRecognitionException, ANTLRException"/>
```

Comment:
please use fully qualified names, collision is unlikely in our project but we need to show other good practices of Check usage